sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Internals/LiveMetricsTagsProcessor.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Monitor.OpenTelemetry.AspNetCore.LiveMetrics.DataCollection;
+using Azure.Monitor.OpenTelemetry.AspNetCore.Models;
+using Azure.Monitor.OpenTelemetry.Exporter.Internals;
+
+namespace Azure.Monitor.OpenTelemetry.AspNetCore.Internals
+{
+    internal struct LiveMetricsTagsProcessor
+    {
+        private static readonly HashSet<string> s_semanticsSet = new()
+        {
+            SemanticConventions.AttributeDbStatement,
+            SemanticConventions.AttributeDbSystem,
+            SemanticConventions.AttributeDbName,
+            SemanticConventions.AttributePeerService,
+
+            // required - HTTP V2
+            SemanticConventions.AttributeHttpRequestMethod,
```

Comment:
@rajkumar-rangaraj - This means we would add other tags like `server.address`, `server.port` etc to the document properties. We don't want that, do we?

Reply:
Correct, we don't want to filter anything for live properties. It's on a best effort first 10 tags are added.